test/analytics/tests/instrumented_events/from_the_browser/viewport_resize.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../services';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const ebtUIHelper = getService('kibana_ebt_ui');
+  const browser = getService('browser');
+  const { common } = getPageObjects(['common']);
+
+  describe('Event "viewport_resize"', () => {
+    beforeEach(async () => {
+      // Navigating to `home` with the Welcome prompt because some runs were flaky
+      // as we handle the Welcome screen only if the login prompt pops up.
+      // Otherwise, it stays in the Welcome screen :/
+      await common.navigateToApp('home');
+    });
+
+    it('should emit a "viewport_resize" event when the browser is resized', async () => {
+      const events = await ebtUIHelper.getEvents(1, {
+        eventTypes: ['viewport_resize'],
+        withTimeoutMs: 100,
+      });
+      expect(events.length).to.be(0);
+      // Resize the window
+      await browser.setWindowSize(500, 500);
+      const { height, width } = await browser.getWindowSize();
+      expect(height).to.eql(500);
+      expect(width).to.eql(500);
```

Comment:
Hum, what are we asserting against here exactly?

Reply:
I initially had `await browser.setWindowSize(100, 100);` and the browser's address bar takes about 125 height. So it was automatically reverting to the default window size.

I added this to validate that the window was correctly resized.